src/controllers/userController.js

```diff
@@ -8,6 +8,18 @@ const {
 } = require('../utils');
 
 const getAllUsers = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch all registered users in a database whose role is user (exclude admins)'
+     #swagger.parameters['role'] = {
+        description: 'Role of the user to fetch',
+     }
+     #swagger.responses[200] = {
+				description: 'Users successfully fetched.',
+        schema: [{ $ref: '#/definitions/PasswordlessUser'' }]
```

Comment:
You have an extra quotation mark here that's causing `swagger-autogen` to print out a warning about this being unparseable when I run the generate api docs command

```suggestion
        schema: [{ $ref: '#/definitions/PasswordlessUser' }]
```